packages/caliper-ethereum/test/ethereum-connector.js

```diff
@@ -0,0 +1,81 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const path = require('path');
+const expect = require('chai').expect;
+const ConfigUtil = require('@hyperledger/caliper-core').ConfigUtil;
+const EthereumConnector = require('../lib/ethereum-connector');
+
+describe('EthereumConnector', function() {
+    let tempConfigFilePath;
+
+    beforeEach(() => {
+        tempConfigFilePath = path.resolve(__dirname, './utils/networkconfig.json');
+        ConfigUtil.set(ConfigUtil.keys.NetworkConfig, tempConfigFilePath);
+    });
+
+
+    describe('EthereumConnector.installSmartContract', () => {
+        it('should throw an error when the specified contract path does not exist', async () => {
+            const invalidConfig = {
+                contracts: {
+                    nonexistent: {
+                        path:  'src/simple/simple.sol'
+                    }
+                }
+            };
+            const ethereumConnector = new EthereumConnector(invalidConfig);
+
+            try {
+                await ethereumConnector.installSmartContract();
+            } catch (err) {
+                expect(err.message).to.contain('Cannot find module');
```

Comment:
`Cannot find module` is not an error I would expect given that the test is supposedly testing installSmartContract with a network configuration file that points to a smart contract file that doesn't exist. Therefore this indicates that there is actually a problem with the implementation. Tests are there to ensure that the code is behaving as expected and in this case I don't think it is but the test would pass, so this test I don't think is correct.